modules/core/arrow-test/src/main/kotlin/arrow/test/laws/ConcurrentLaws.kt

```diff
@@ -711,8 +711,8 @@ object ConcurrentLaws {
 
       just(Unit).flatMap {
         raiseError<Unit>(RuntimeException("failed"))
-          .onError(incrementCounter)
-      }.onError(incrementCounter)
+          .onError { incrementCounter }
```

Comment:
What is the point of such a change? `incrementCounter` is just a `val`, looks like this makes no sense.